### DISCUSSION TITLE
Decimal Attribute Validation

### DISCUSSION THREAD

**Discussion by wit3** [spatie/laravel-data#421] [Q&A] [answered]
I don't have a complete overview to create a validation for decimal types like the original from doc

**#[decimal:min,max](https://laravel.com/docs/10.x/validation#rule-decimal)**
```php
// Must have exactly two decimal places (9.99)...
'price' => 'decimal:2'
 
// Must have between 2 and 4 decimal places...
'price' => 'decimal:2,4'
```

it's missing in new Attribute Validation class ( ex: Validate(min,max) )

**Comment by Nir-An** [comment, marked as answer]
Hi @wit3  :+1: 

you're always can use the [`rule` attribute ](https://spatie.be/docs/laravel-data/v3/as-a-data-transfer-object/request-to-data-object#content-rule-attribute)

```php
#[Rule(['required', 'decimal:2,4'])] 
public float $price
```

**Reply by wit3** [reply]
Dude you've right 👍

**Reply by Nir-An** [reply]
@wit3 

my pleasure :1st_place_medal: 
it is very helpful for others, if you change it from idea to Q&A and sign it as resolved answer
